translations/dutch.json

```diff
@@ -39,9 +38,9 @@
 		"DATA COMMANDS": "DATA COMMANDO'S",
 		"DRIVER COMMANDS": "DRIVER COMMANDO'S",
 		"MODERATOR COMMANDS": "MODERATOR COMMANDO'S",
-		"LEADER COMMANDS": "LEADER COMMANDO'S",
+		"ADMIN COMMANDS": "LEADER COMMANDO'S",
```

Comment:
The "LEADER COMMANDO'S" should be "ADMIN COMMANDO'S"?

Reply:
Yes — this pull request removes the line ``` "LEADER COMMANDS": "LEADER COMMANDO'S", ```, replacing it with ``` "ADMIN COMMANDS": "ADMINISTRATOR COMMANDO'S", ```. Perhaps you were looking at outdated code?

Reply:
Ah, my apologies